app/assets/stylesheets/posts.scss

```diff
@@ -3,32 +3,42 @@
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
 
-
 #posts-home{
+	    display: flex;
+    margin-top: 20px;
+    justify-content: space-between;
+#GL{
 	
-	display: flex;
-	justify-content: space-evenly;
-	padding-top: 25px;
-
-	.deewar-posts{
-		width: 50%;
-		border-right: 1px dashed #007effe0;
-		border-left: 1px dashed #007effe0;
-
-		.new-post-form{
-			display: flex;
-			justify-content: center;
-			form{
-			display: flex;
-				flex-direction: column;
-			}
-			
-		}
-
-		.posts-list{
-			
-		}
-	}
+	box-shadow: 11px 28px 82px 19px #21b4ec;
+    /* width: 15vh; */
+    /* background-color: white; */
+    flex-basis: 8vw;
+}
+#FL{
+	width:8vw;
+	box-shadow: 40px 14px 40px 23px #21b4ec;
+}
```

Comment:
**Suggestion:** Avoid using magic numbers in your CSS. For example, the box-shadow values seem arbitrary. Consider defining these values as variables at the top of your file, or in a separate configuration file, to make it clear what they represent and to make them easier to change in the future.
```suggestion
 $shadow-color: #21b4ec;
 $shadow-GL: 11px 28px 82px 19px;
 $shadow-FL: 40px 14px 40px 23px;

 #GL{
 	box-shadow: $shadow-GL $shadow-color;
 	flex-basis: 8vw;
 }
 #FL{
 	width:8vw;
 	box-shadow: $shadow-FL $shadow-color;
 }
```